### PR TITLE
Add favicon and homescreen icons to templates (fixes #1449)

### LINKED ISSUE #1449: Give templates favicons & homescreen icons
[xref](https://github.com/google/material-design-lite/pull/1418#issuecomment-133354753)

### PR DESCRIPTION
**Pull request by surma**

### CODE REVIEW

File: templates/article/index.html [hunk 1]
```diff
@@ -25,18 +25,20 @@
...
     <meta name="apple-mobile-web-app-status-bar-style" content="black">
     <meta name="apple-mobile-web-app-title" content="Material Design Lite">
-    <link rel="apple-touch-icon-precomposed" href="apple-touch-icon-precomposed.png">
+    <link rel="apple-touch-icon-precomposed" href="images/ios-desktop.png">
 
     <!-- Tile icon for Win8 (144x144 + tile color) -->
     <meta name="msapplication-TileImage" content="images/touch/ms-touch-icon-144x144-precomposed.png">
     <meta name="msapplication-TileColor" content="#3372DF">
 
+    <link rel="shortcut icon" href="images/favicon.png" />
```
Comment (sindresorhus) [inline templates/article/index.html]:
```html
<link rel="shortcut icon" href="images/favicon.png">
```

File: templates/blog/index.html [hunk 2]
```diff
@@ -25,18 +25,20 @@
...
     <meta name="apple-mobile-web-app-status-bar-style" content="black">
     <meta name="apple-mobile-web-app-title" content="Material Design Lite">
-    <link rel="apple-touch-icon-precomposed" href="apple-touch-icon-precomposed.png">
+    <link rel="apple-touch-icon-precomposed" href="images/ios-desktop.png">
 
     <!-- Tile icon for Win8 (144x144 + tile color) -->
     <meta name="msapplication-TileImage" content="images/touch/ms-touch-icon-144x144-precomposed.png">
     <meta name="msapplication-TileColor" content="#3372DF">
 
+    <link rel="shortcut icon" href="images/favicon.png" />
```
Comment (sindresorhus) [inline templates/blog/index.html]:
```html
<link rel="shortcut icon" href="images/favicon.png">
```

File: templates/dashboard/index.html [hunk 3]
```diff
@@ -25,18 +25,20 @@
...
     <meta name="apple-mobile-web-app-status-bar-style" content="black">
     <meta name="apple-mobile-web-app-title" content="Material Design Lite">
-    <link rel="apple-touch-icon-precomposed" href="apple-touch-icon-precomposed.png">
+    <link rel="apple-touch-icon-precomposed" href="images/ios-desktop.png">
 
     <!-- Tile icon for Win8 (144x144 + tile color) -->
     <meta name="msapplication-TileImage" content="images/touch/ms-touch-icon-144x144-precomposed.png">
     <meta name="msapplication-TileColor" content="#3372DF">
 
+    <link rel="shortcut icon" href="images/favicon.png" />
```
Comment (sindresorhus) [inline templates/dashboard/index.html]:
```html
<link rel="shortcut icon" href="images/favicon.png">
```

File: templates/text-only/index.html [hunk 4]
```diff
@@ -25,18 +25,20 @@
...
     <meta name="apple-mobile-web-app-status-bar-style" content="black">
     <meta name="apple-mobile-web-app-title" content="Material Design Lite">
-    <link rel="apple-touch-icon-precomposed" href="apple-touch-icon-precomposed.png">
+    <link rel="apple-touch-icon-precomposed" href="images/ios-desktop.png">
 
     <!-- Tile icon for Win8 (144x144 + tile color) -->
     <meta name="msapplication-TileImage" content="images/touch/ms-touch-icon-144x144-precomposed.png">
     <meta name="msapplication-TileColor" content="#3372DF">
 
+    <link rel="shortcut icon" href="images/favicon.png" />
```
Comment (sindresorhus) [inline templates/text-only/index.html]:
```html
<link rel="shortcut icon" href="images/favicon.png">
```

**Comment by surma** [comment]
@sindresorhus You are right! I’d addressing that in a separate PR though, as we have that kind XHTML thing in more places in the microsite.

**Comment by addyosmani** [comment]
LGTM for this specific set of changes but I agree with Sindre - let's tackle the suggested edits in your other PR.

**Pull request closed without merging**